src/api/services/booksService.test.tsx

```diff
@@ -0,0 +1,139 @@
+import { bookService } from './booksService';
+import { booksApi } from '../axios';
+import { vi } from 'vitest';
+
+vi.mock('../axios', () => ({
+  booksApi: {
+    get: vi.fn(),
+  },
+}));
+
+describe('BooksService', () => {
+  it('Returns list of books for valid query', async () => {
+    const mockResponse = {
```

Comment:
```suggestion
describe('BooksService', () => {
  describe('getBooksList', () => {
    it('Returns list of books for valid query', async () => {
      const mockResponse = {
```

все эти тесты относятся к методу getBooksList, просто пока он только один, кажется, что это тесты всей апихи